scalapact-shared/src/test/scala/com/itv/scalapact/shared/utils/HelpersSpec.scala

```diff
@@ -19,6 +19,12 @@ class HelpersSpec extends FunSpec with Matchers {
 
     }
 
+    it("should parse various date strings") {
+      Helpers.safeStringToDateTime("2020-08-06").isDefined shouldBe true
```

Comment:
could be more strict with the assertions? like `shouldBe Some(OffsetDateTime.of(2020, 8, 6, ...))`

Reply:
but Daaaaan, I'm lazy


sure can do

Reply:
😆